SwiftExercises.playground/section-1.swift

```diff
@@ -47,23 +61,33 @@ let worf = [
     "favorite drink": "prune juice",
     "quote" : "Today is a good day to die."]
 
+
+
 let picard = [
     "name": "Jean-Luc Picard",
     "rank": "captain",
     "information": "Captain of the USS Enterprise",
     "favorite drink": "tea, Earl Grey, hot"]
 
+
 let characters = [worf, picard]
 
+
 func favoriteDrinksArrayForCharacters(characters:Array<Dictionary<String, String>>) -> Array<String> {
     // return an array of favorite drinks, like ["prune juice", "tea, Earl Grey, hot"]
     // WORK HERE
-    return []
+    var favDrinks = [String]() // Don't understand why I got to put ()
+    for character in characters {
+  //      favDrinks += [character["favorite drinks"]] // why does this not work?
```

Comment:
This doesn't work because character["favorite drinks"] returns an optional (since the element might not have this value).